src/toolbars/toolbar-component.js

```diff
@@ -52,6 +52,7 @@ angular.module('patternfly.toolbars').component('pfToolbar', {
 
     function viewSelected (viewId) {
       ctrl.config.viewsConfig.currentView = viewId;
+      ctrl.config.isTableView = viewId === 'tableView';
```

Comment:
Removed 'isTableView' and are using the existing viewConfig.currentView.   Weird but in `setupConfig()` the `ctrl.config.viewsConfig.currentView` was being set to the view object; whereas elsewhere in the component, and in the unit tests, it is being set to an id.